store/file/ods.go

```diff
@@ -108,22 +108,45 @@ func writeODSFile(w io.Writer, eds *rsmt2d.ExtendedDataSquare, axisRoots *share.
 		return nil, fmt.Errorf("writing axis roots: %w", err)
 	}
 
-	for _, shr := range eds.FlattenedODS() {
-		if _, err := w.Write(shr); err != nil {
-			return nil, fmt.Errorf("writing shares: %w", err)
-		}
+	// write quadrants
+	err = writeQuadrant(w, eds, 0)
+	if err != nil {
+		return nil, fmt.Errorf("writing Q1: %w", err)
 	}
 	return h, nil
 }
 
+func writeQuadrant(w io.Writer, eds *rsmt2d.ExtendedDataSquare, quadrantIdx int) error {
```

Comment:
It would be nice to doc that it writes row by row with a buffer. with a size of a row